service/src/test/java/bio/terra/workspace/connected/WorkspaceConnectedTestUtils.java

```diff
@@ -1,11 +1,16 @@
 package bio.terra.workspace.connected;
 
 import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doReturn;
 
 import bio.terra.workspace.service.iam.AuthenticatedUserRequest;
 import bio.terra.workspace.service.iam.model.SamConstants;
 import bio.terra.workspace.service.job.JobService;
 import bio.terra.workspace.service.spendprofile.SpendConnectedTestUtils;
+import bio.terra.workspace.service.spendprofile.SpendProfile;
```

Comment:
nit: it looks like you're not using any of the new imports, remove?
I'm surprised spotless didn't clean these up

Reply:
Spotless appears to be not working on the test source set anymore. I might have broken that.